### PR TITLE
Intezer v2

### PR DESCRIPTION
**Pull request by DeanArbel**
## Status
In Progress

## Related Issues
fixes: https://github.com/demisto/etc/issues/16004

## Description
Intezer v2 implementation with corresponding playbooks and relevant scripts.

## Screenshots
image

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [x] Documentation (https://github.com/demisto/etc/issues/17591)
- [ ] Code Review

### CODE REVIEW

**Comment by kirbles19** [comment]
All .yml files updated, all done.

File: Integrations/IntezerV2/IntezerV2.py [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+    return_error(str(e))
+
+
+def main():
+    try:
+        handle_proxy()
+        if demisto.command() == 'test-module':
+            demisto.results(check_is_available())
+        elif demisto.command() == 'intezer-analyze-by-hash':
+            analyze_by_hash()
```
Comment (yuvalbenshalom) [inline Integrations/IntezerV2/IntezerV2.py]: use `sample_func_command()` naming

File: Integrations/IntezerV2/IntezerV2.py [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+
+''' HELPER FUNCTIONS '''
+
+
+def handle_response(response, acceptable_http_status_codes):
+    if response.status_code not in acceptable_http_status_codes:
+        error_msg = http_status_to_error_massage.get(response.status_code, "Failed to perform request")
+        return_error(f'{ERROR_PREFIX} {error_msg}')
+
+    return response.json()
```
Comment (yuvalbenshalom) [inline Integrations/IntezerV2/IntezerV2.py]: wrap this with try and except exception for failed to parse json, and return appropriate error

File: Scripts/script-IntezerRunScanner.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+- agent
+- forensics
+comment: Running the Intezer Endpoint Analysis Scanner
+enabled: true
+args:
+- name: api_key
+  required: true
+  secret: true
+  description: 'The Intezer API key '
+- name: endpoint_analysis_id
```
Comment (yuvalbenshalom) [inline Scripts/script-IntezerRunScanner.yml]: what is the meaning of sending analysis id to the scanner on the end point?

**Pull request merged**